examples/module_mock/__tests__/partial_mock.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
```

Comment:
What copyright header should I use? I got this header from the other example projects:)